spark/src/main/scala/io/delta/tables/execution/DeltaTableOperations.scala

```diff
@@ -112,6 +113,41 @@ trait DeltaTableOperations extends AnalysisHelper { self: DeltaTable =>
     toDataset(sparkSession, restore)
   }
 
+  protected def executeClone(
+    table: DeltaTableV2,
+    target: String,
+    replace: Boolean,
+    tableProperties: Map[String, String],
```

Comment:
```suggestion
    properties: Map[String, String],
```

Let's move it to the end of this function to be coherent with the clone Scala APIs as well

Reply:
You mean after `versionAsOf` and `timestampAsOf` that are optional with default values?